Gulpfile.js

```diff
@@ -376,12 +376,42 @@ gulp.task('build', DEV_MODE ? gulp.registry().get('fast-build') : gulp.parallel(
 // Test
 gulp.step('prepare-tests', gulp.registry().get(SKIP_BUILD ? 'lint' : 'build'));
 
+function exitDomains () {
```

Comment:
Move the `exitDomains` and `enterDomains` functions to the `gulp/helpers/domain.js` file.
Also add the `!gulp` statement to the `.gitignore` file - https://github.com/DevExpress/testcafe/pull/5724/files#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5R3